mteb/tasks/Clustering/eng/MedrxivClusteringS2S.py

```diff
@@ -1,11 +1,50 @@
 from __future__ import annotations
 
+from mteb.abstasks.AbsTaskClustering import AbsTaskClustering
+from mteb.abstasks.AbsTaskClusteringFast import AbsTaskClusteringFast
 from mteb.abstasks.TaskMetadata import TaskMetadata
 
-from ....abstasks.AbsTaskClustering import AbsTaskClustering
+
+class MedrxivClusteringS2SFast(AbsTaskClusteringFast):
+    metadata = TaskMetadata(
+        name="MedrxivClusteringS2SFast",
```

Comment:
I would name it as such:
```suggestion
        name="MedrxivClusteringS2S.v2",
```
same for the others